src/main/java/com/eu/habbo/habbohotel/rooms/RoomManager.java

```diff
@@ -808,8 +809,9 @@ public void enterRoom(final Habbo habbo, final Room room) {
             allFloorItems.forEach(new TObjectProcedure<HabboItem>() {
                 @Override
                 public boolean execute(HabboItem object) {
-                    if (room.isHideWired() && object instanceof InteractionWired)
+                    if ((room.isHideWired() && object instanceof InteractionWired) || object.isItemHiddenByAreaHider()) {
```

Comment:
Dont fix now: perhaps we should condense these checks into a single `object.isHidden()` but we can do this later. Otherwise this PR might become too big.